src/app/files/files.component.html

```diff
@@ -68,7 +68,13 @@
       [matRippleUnbounded]="false"
       class="files__object"
     >
-      <img src="assets/object.svg" class="files__icon" />
+      <div class="files__icon_wrapper {{file.successfull}}">
```

Comment:
Can we use `ngClass` for the `file.successfull`? https://angular.io/api/common/NgClass